File: property/src/main/java/com/jingxi/smartlife/pad/property/mvp/view/SimpleFixFragment.java
package com.jingxi.smartlife.pad.property.mvp.view;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;

import com.jingxi.smartlife.pad.property.R;
import com.wujia.lib_common.base.BaseFragment;

/**
 * author ：shenbingkai@163.com
 * date ：2019-02-17
 * description ：
 */
public class SimpleFixFragment extends BaseFragment implements View.OnClickListener {


    public SimpleFixFragment() {

    }

    public static SimpleFixFragment newInstance() {
        SimpleFixFragment fragment = new SimpleFixFragment();
        Bundle args = new Bundle();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    protected int getLayoutId() {
        return R.layout.fragment_propery_fix_simple;
    }

    @Override
    public void onLazyInitView(@Nullable Bundle savedInstanceState) {
        super.onLazyInitView(savedInstanceState);

        $(R.id.fix_electricity_btn).setOnClickListener(this);
        $(R.id.fix_water_btn).setOnClickListener(this);
        $(R.id.fix_public_btn).setOnClickListener(this);

    }

    @Override
    public void onClick(View v) {
        if (v.getId() == R.id.fix_electricity_btn) {

        } else if (v.getId() == R.id.fix_water_btn) {

        } else if (v.getId() == R.id.fix_public_btn) {

        }
    }
}

File: property/src/main/java/com/jingxi/smartlife/pad/property/mvp/view/SimpleFixFragment.kt
package com.jingxi.smartlife.pad.property.mvp.view

import android.os.Bundle
import android.view.View

import com.jingxi.smartlife.pad.property.R
import com.wujia.lib.widget.util.ToastUtil
import com.wujia.lib_common.base.BaseFragment


/**
 * author ：shenbingkai@163.com
 * date ：2019-02-17
 * description ：
 */
class SimpleFixFragment : BaseFragment(), View.OnClickListener {

    override fun getLayoutId(): Int {
        return R.layout.fragment_propery_fix_simple
    }

    override fun onLazyInitView(savedInstanceState: Bundle?) {
        super.onLazyInitView(savedInstanceState)

        `$`<View>(R.id.fix_electricity_btn).setOnClickListener(this)
        `$`<View>(R.id.fix_water_btn).setOnClickListener(this)
        `$`<View>(R.id.fix_public_btn).setOnClickListener(this)
        `$`<View>(R.id.fix_call_service_btn).setOnClickListener(this)

    }

    override fun onClick(v: View) {
        if (v.id == R.id.fix_electricity_btn) {

        } else if (v.id == R.id.fix_water_btn) {

        } else if (v.id == R.id.fix_public_btn) {

        } else if (v.id == R.id.fix_call_service_btn) {
            ToastUtil.showShort(mContext, getString(R.string.not_join))
        }
    }

    companion object {

        fun newInstance(): SimpleFixFragment {
            val fragment = SimpleFixFragment()
            val args = Bundle()
            fragment.arguments = args
            return fragment
        }
    }
}
